recipes/climtools/25.2/gh200/environments.yaml

```diff
@@ -0,0 +1,28 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.30
+      gpu: false
+  unify: true
+  specs:
+  # Info about versions for later update:
+  # - the eccodes version is pinned to 2.36.4 to match the icon/25.2 recipe
+  # - the python minor version is set to the latest possible spack accepts
+  # - other versions where just picked by the spack concretizer
+  - hdf5 +szip +hl  # concretized to @1.14.3
+  - netcdf-c  # concretized to @4.9.2
```

Comment:
The comments are confusing, soon this will be a copy-paste artifact that is wrong.

I  would put version where we care:  netcdf-c, netcdf-fortran and eccodes.
All other spec can live w/o. So also no comments.

Reply:
Fine, I wasn't 100% convinced myself. Why do we care about the `netcdf-c` version (there's no `netcdf-fortran)`? It's just the backend used by python packages or other applications like cdo or nco. Or do you plan to build specific applications with it?

Reply:
But please remove the comments stating the version.